genestack_client/genestack_exceptions.py

```diff
@@ -78,14 +78,17 @@ def __init__(self, my_version, compatible=None):
         :type compatible: distutils.version.StrictVersion
         """
         if compatible:
```

Comment:
As I understand now, this is an unfortunate argument name.
Should be `current_version` and `required_version`.

Moreover, it's weird to have to pass current version of Python Client to throw this exception.